docs/src/administration/users.md

```diff
@@ -72,67 +60,27 @@ Available commands:
 * `platform user:role`
   * View or change a user's role
 
-For example, the following command would add `alice@example.com` with **Project Admin** role in the current project.
+ **Example**
+
+ Add `user1@example.com` to the project with Project Admin permissions:
 
 ```bash
-platform user:add alice@example.com -r admin
+platform user:add user1@example.com -r admin
 ```
+Once you add the user to the project, they receive an invitation email to confirm their details and a registration link.
```

Comment:
This is what I meant above about the outcome. So just needs to be moved there. Also, it's not true for people who are already Platform users -- they just get an invitation to the project.